t/python/t0005-kvs.py

```diff
@@ -114,9 +114,10 @@ def test_fill(self):
             kd.fill({"things": 1, "stuff": "strstuff", "dir.other_thing": "dirstuff"})
```

Comment:
Commit message for "testsuite: check kvs writes with new context ":  an extra "the"